README.md

```diff
@@ -2,3 +2,5 @@
 my demo repository where I'm following the help directions
 
 I triiiied to make a sandbox repository without having read or followed these instructions (although I do really like the youtube video I was following). And then I ran into trouble! So here I am. 
+
+Now I'm adding some stuff to this file so that I can play around with the behavior in the diff and make some suggestions and commnets. 
```

Comment:
```suggestion
Now I'm adding some stuff to this file so that I can play around with the behavior in the diff and make some suggestions and comments. 
```

And I can add a comment explaining the suggested edit, so the writer can decide whether to accept or reject it.